README.md

```diff
@@ -1,16 +1,9 @@
-# veryrezsi
+# 💲 VeryRezsi 💲
+###### Status
+![Docker Build](https://github.com/szattila98/veryrezsi/actions/workflows/images.yml/badge.svg)
```

Comment:
Hella cool, this updates itself based on the statuses I presume.

Reply:
Sure it does. We need a lot of badges to be real GH users.